tests/features/data-visibility-and-review.feature

```diff
@@ -1,122 +1,202 @@
-#    And retrieving the dataset via the api should return visibility "public" and review_status "provisional"
-#
-#  @skip
-#  @patch
-#  Scenario: Update review status for a new dataset without changing visibility
-#    Given the system has a new dataset with visibility "internal" and review_status "provisional"
-#    When an authorized reviewer updates the dataset review_status to "approved" via the api
-#    Then the system should persist the dataset with review_status "approved"
-#    And the system should preserve the visibility as "internal"
-#    And retrieving the dataset via the api should return visibility "internal" and review_status "approved"
-#
-#  @skip
-#  @authorization
-#  Scenario: Allow all datasets for internal users
-#    Given the system has datasets with combinations of:
-#      | visibility | review_status |
-#      | internal   | provisional   |
-#      | internal   | approved      |
-#      | public     | provisional   |
-#      | public     | approved      |
-#    And the caller is identified as an internal staff user
-#    When the internal staff user requests those datasets via the api
-#    Then the system should return all of those datasets in the response
-#
-#  @skip @authorization
-#  Scenario: Reject unauthorized changes to visibility or review status
-#    Given the system has a dataset with visibility "internal" and review_status "provisional"
-#    And the caller is authenticated without permission to change visibility or review_status
-#    When the caller attempts to update the visibility to "public" via the api
-#    And the caller attempts to update the review_status to "approved" via the api
-#    Then the system should return an authorization error for these updates
-#    And the dataset should remain persisted with visibility "internal" and review_status "provisional"
-#
-#  @skip
-#  @disclaimer
-#  Scenario: Include disclaimer information based on review status
-#    Given the system has a dataset with visibility "public" and review_status "provisional"
-#    When any authorized caller retrieves the dataset via the api
-#    Then the system should return a response in JSON format
-#    And the response should include a disclaimer field derived from the review_status
-#    And the disclaimer should indicate that the data is provisional
-#
-#    And the system has a dataset with visibility "internal" and review_status "approved"
-#    When any authorized caller retrieves that dataset via the api
-#    Then the response should include a disclaimer field derived from the review_status
-#    And the disclaimer should indicate that the data is approved
-#
+# Created by marissa at 12/11/2025
+# file: tests/features/data_visibility_and_review_backend.feature
+
+@backend @BDMS-XXX
+Feature: Manage data visibility separately from review in the backend
+  As an Ocotillo data manager
+  I want visibility and review to be stored as two separate backend attributes
```

Comment:
Should this read `...and review status...` to reflect the verbiage used later on?